### ISSUE TITLE
hummingbot_quickstart.py doesn't work with the password parameter/environment variable. 

### ISSUE THREAD

**Issue by stalkopat** [hummingbot/hummingbot#5036]
**Describe the bug**
If a password is provided with either the -p argument or by using the CONFIG_PASSWORD environment variable quickstart fails with the following Stacktrace:

> File "c:\<Path>\hummingbot\bin\hummingbot_quickstart.py", line 161, in <module>
>     main()
>   File "c:\<Path>\hummingbot\bin\hummingbot_quickstart.py", line 157, in main
>     asyncio.get_event_loop().run_until_complete(quick_start(args))
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\asyncio\base_events.py", line 616, in run_until_complete
>     return future.result()
>   File "c:\<Path>\hummingbot\bin\hummingbot_quickstart.py", line 136, in quick_start
>     await safe_gather(*tasks)
>   File "C:\<Path>\hummingbot\hummingbot\core\utils\async_utils.py", line 22, in safe_gather
>     return await asyncio.gather(*args, **kwargs)
>   File "C:\<Path>\hummingbot\hummingbot\client\hummingbot_application.py", line 213, in run
>     await self.app.run()
>   File "C:\<Path>\hummingbot\hummingbot\client\ui\hummingbot_cli.py", line 91, in run
>     await self.app.run_async()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\application\application.py", line 816, in run_async
>     return await _run_async2()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\application\application.py", line 792, in _run_async2
>     result = await _run_async()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\application\application.py", line 728, in _run_async
>     self._redraw()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\application\application.py", line 552, in _redraw
>     self.context.copy().run(run_in_context)
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\application\application.py", line 535, in run_in_context
>     self.renderer.render(self, self.layout)
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\renderer.py", line 707, in render
>     output.flush()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\output\windows10.py", line 53, in flush
>     self.vt100_output.flush()
>   File "C:\<SecondPath>\miniconda3\envs\hummingbot\lib\site-packages\prompt_toolkit\output\vt100.py", line 691, in flush
>     out.write(data.encode(self.stdout.encoding or "utf-8", "replace"))
>   File "C:\<Path>\hummingbot\hummingbot\client\ui\stdout_redirection.py", line 89, in write
>     self._write(data)
>   File "C:\<Path>\hummingbot\hummingbot\client\ui\stdout_redirection.py", line 70, in _write
>     if '\n' in data:
> TypeError: a bytes-like object is required, not 'str'
> 

**Steps To Reproduce**
Launch hummingbot_quickstart.py with the -p parameter or a CONFIG_PASSWORD environment variable setup.

**Release version**
0.46, should affect development as well

**Underlying problem**
From my Debugging it seems like no Application() constructor of the prompt_toolkit is called unless a login_prompt() or another dialog is called before Hummingbot CLI is started. This can be tested by adding the following into hummingbot_quickstart:

```
from prompt_toolkit.styles import Style
from prompt_toolkit.application import Application
from prompt_toolkit.key_binding.bindings.focus import focus_next, focus_previous
from prompt_toolkit.key_binding.defaults import load_key_bindings
from prompt_toolkit.key_binding.key_bindings import KeyBindings, merge_key_bindings

style = Style.from_dict({
    'dialog': 'bg:#171E2B',
    'dialog frame.label': 'bg:#ffffff #000000',
    'dialog.body': 'bg:#000000 #1CD085',
    'dialog shadow': 'bg:#171E2B',
    'button': 'bg:#000000',
    'text-area': 'bg:#000000 #ffffff',
})
```

and adjusting the main() function as follows:
```
def main():
    args = CmdlineParser().parse_args()

    # Parse environment variables from Dockerfile.
    # If an environment variable is not empty and it's not defined in the arguments, then we'll use the environment
    # variable.
    if args.config_file_name is None and len(os.environ.get("CONFIG_FILE_NAME", "")) > 0:
        args.config_file_name = os.environ["CONFIG_FILE_NAME"]
    if args.wallet is None and len(os.environ.get("WALLET", "")) > 0:
        args.wallet = os.environ["WALLET"]
    if args.config_password is None and len(os.environ.get("CONFIG_PASSWORD", "")) > 0:
        args.config_password = os.environ["CONFIG_PASSWORD"]

    # If no password is given from the command line, prompt for one.
    if args.config_password is None:
        if not login_prompt():
            return
    else:
        bindings = KeyBindings()
        bindings.add("tab")(focus_next)
        bindings.add("s-tab")(focus_previous)

        Application(
            key_bindings=merge_key_bindings([load_key_bindings(), bindings]),
            mouse_support=True,
            style=style,
            full_screen=True,
        )

    asyncio.get_event_loop().run_until_complete(quick_start(args))
```

This fix obviously isn't very clean and really should be handled somewhere in the hummingbot start function.

**Comment by martinkou** [comment]
This is being fixed with #5109

**Comment by JeremyKono** [comment]
The fix has been merged into `development` and `staging. Closing this issue.
